pkg/cmd/root.go

```diff
@@ -9,6 +9,8 @@ func NewRootCommand() *cobra.Command {
 		Use:   "openfga",
 		Short: "A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar",
 		Long: `A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar.
-OpenFGA is designed to make it easy for developers to model their application permissions, and to add and integrate fine-grained authorization into their applications.`,
+
+OpenFGA is designed to make it easy for developers to model their application permissions, and to add and integrate fine-grained authorization into their applications.
+Complete documentation is available at https://openfga.dev`,
```

Comment:
Should this sentence end with a period?

Reply:
Yeah - I just pushed a commit to add it.